app/controllers/activitypub/collections_controller.rb

```diff
@@ -12,25 +12,17 @@ class ActivityPub::CollectionsController < ActivityPub::BaseController
 
   def show
     expires_in 3.minutes, public: public_fetch_mode?
-    render_with_cache json: collection_presenter, content_type: 'application/activity+json', serializer: ActivityPub::CollectionSerializer, adapter: ActivityPub::Adapter, skip_activities: true
+    render_with_cache json: collection_presenter, content_type: 'application/activity+json', serializer: ActivityPub::CollectionSerializer, adapter: ActivityPub::Adapter
```

Comment:
`skip_activities` doesn't do anything anymore? when have we removed it?

Reply:
I can't even find where it was used... It doesn't seem to appear anywhere else in the code.